index.src.html

```diff
@@ -519,17 +525,23 @@ <h3 id="referrer-policy-header">Delivery via Referrer-Policy header</h3>
   made, and with
   <a for=/>browsing contexts</a> created from the context of the protected resource.
   The syntax for the name and value of the header are described by the
-  following ABNF grammar:
+  following <a>ABNF</a> grammar:
 
   <pre>
-    "Referrer-Policy:" 1#<a>policy-token</a>
+    "Referrer-Policy:" 1#(<a>policy-token</a> / <a>extension-token</a>)
   </pre>
   <pre>
-    <dfn export>policy-token</dfn>   = "no-referrer" / "no-referrer-when-downgrade" / "strict-origin" / "strict-origin-when-cross-origin" / "same-origin" / "origin" / "origin-when-cross-origin" / "unsafe-url"
+    <dfn export>policy-token</dfn> = "no-referrer" / "no-referrer-when-downgrade" / "strict-origin" / "strict-origin-when-cross-origin" / "same-origin" / "origin" / "origin-when-cross-origin" / "unsafe-url"
```

Comment:
Nit: This should probably be defined as a `grammar` type. The simplest way to do that is to add `dfn-type="grammar"` to the `<pre>` wrapping the ABNF bits.